core-services/prow/02_config/_plugins.yaml

```diff
@@ -292,6 +292,12 @@ config_updater:
         default:
         - ""
       name: ci-operator-misc-configs
+    ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-feature*.yaml:
```

Comment:
you don't have to do that. Please remove it

Reply:
after debugging with @hongkailiu we found that the reason i was getting prow config failures [1] was the lack of definition in this file... this change is for a new feature branch so none of the existing patterns matched the new job and config file pattern.

[1] https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6267/pull-ci-openshift-release-master-correctly-sharded-config/10124